### DISCUSSION TITLE
[Feature] Allow Ctrl-click / Cmd-click on photos (web)

### DISCUSSION THREAD

**Discussion by erikvanoosten** [immich-app/immich#9908] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In the web timeline, please allow Ctrl-click / Cmd-click on a photo so that it can be opened in another tab.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Now that the asset viewer has a proper fixed URL, this should be relatively straightforward to implement by making the thumbnails in the timeline an actual link rather than a div with a click event. As a bonus that's also better for accessibility.

**Reply by GeorgeShao** [reply]
I've pushed this change in https://github.com/immich-app/immich/pull/9954

It ended up being a bit more complicated than expected (not simply just replacing `<div>` with `<a>` and the click event with href), because we need to support selecting one or more photos.

I spent quite some time ensuring the behaviour of my code (including photo clicking, selection, deselection) matches the current release behaviour, so the only difference should be the new ability to ctrl/cmd+click (depending on operating system) and better accessibility from using the `<a>` tag.
